back/src/main/kotlin/com/back/boundedContexts/post/application/service/PostApplicationService.kt

```
@@ -281,14 +241,14 @@ class PostApplicationService(
                 }
             post.modify(title, content, published, listed, sanitizedContentHtml)
             postRepository.flush()
-            syncMetaTagIndexAttr(post)
+            postTagIndexService.syncMetaTagIndexAttr(post)
             if (wasTempDraft) {
-                updateTempDraftMarker(post.author, null)
+                postTempDraftService.updateTempDraftMarker(post.author, null)
             }
         } catch (exception: ObjectOptimisticLockingFailureException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)